site/src/pages/index.module.css

```diff
@@ -58,5 +62,14 @@
 
   .cardContainer {
     flex-direction: column;
+    padding: calc(var(--uitk-size-unit) * 3) calc(var(--uitk-size-unit) * 2);
+    gap: calc(var(--uitk-size-unit) * 2);
+  }
+}
+
+@media screen and (max-width: 1280px) {
+  .heroTitle {
+    --ifm-heading-line-height: var(--uitk-text-h1-lineHeight);
+    --ifm-h1-font-size: var(--uitk-text-h1-fontSize);
```

Comment:
You could move this above the 996px query just for consistency / readability but not a biggie

Reply:
I've tried this but I think it may be best to leave it as is because otherwise the hero title looks quite massive when we change to touch density
<img width="1045" alt="Screenshot 2022-12-05 at 10 17 42" src="https://user-images.githubusercontent.com/31695437/205612606-ca215b44-ed47-4138-9990-b0e5a2acea3d.png">